backup/statistics.go

```diff
@@ -14,15 +14,23 @@ import (
 	"github.com/lib/pq"
 )
 
+func printTupleStatisticsStatementForTable(statisticsFile *utils.FileWithByteCount, tocfile *toc.TOC, table Table, tupleStat TupleStatistic) {
+	tupleQuery := GenerateTupleStatisticsQuery(table, tupleStat)
+	printStatisticsStatementForTable(statisticsFile, tocfile, table, tupleQuery)
+}
+
+func printAttributeStatisticsStatementForTable(statisticsFile *utils.FileWithByteCount, tocfile *toc.TOC, table Table, attStat AttributeStatistic) {
+	attributeQueries := GenerateAttributeStatisticsQueries(table, attStat)
+	for _, attrQuery := range attributeQueries {
+		printStatisticsStatementForTable(statisticsFile, tocfile, table, attrQuery)
+	}
+}
+
 func PrintStatisticsStatements(statisticsFile *utils.FileWithByteCount, tocfile *toc.TOC, tables []Table, attStats map[uint32][]AttributeStatistic, tupleStats map[uint32]TupleStatistic) {
```

Comment:
The PrintStatisticsStatements function is used in the tests only. Can we delete the function?

Reply:
If 

> we delete the function

then we don't test printStatisticsStatementForTable

Reply:
Let's move the function to tests

Reply:
moved